config-linux.md

```diff
@@ -147,7 +147,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/random`][random.4]
 * [`/dev/urandom`][random.4]
 * [`/dev/tty`][tty.4]
-* [`/dev/console`][console.4]
+* [`/dev/console`][console.4] is setup if terminal is enabled in the config by bind mounting the psuedoterminal slave to /dev/console.
```

Comment:
I think `/dev/console` can be a common terminal and not just pseudoterminal, no?